### PR TITLE
add the Unattended-Upgrade::MailReport setting to unattended-upgrades.erb

### PR DESCRIPTION
**Pull request by kenyon**
#### Pull Request (PR) description
Add the ability to configure the Unattended-Upgrade::MailReport setting. See https://github.com/mvo5/unattended-upgrades/blob/master/README.md for setting reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,8 @@ altering some of the default settings.
   xenial, bionic and default), `false` for Debian (wheezy and default)):
   Use the legacy `Unattended-Upgrade::Allowed-Origins` setting or the modern `Unattended-Upgrade::Origins-Pattern`.
 * `mail`: A hash to configure email behaviour with two possible keys:
```
Comment (alexjfisher) [inline README.md:114]: ‘Two possible keys’?

**Comment by alexjfisher** [comment]
@ghoneycutt Thanks for reviewing, but could you be a bit less trigger happy with the merge button? 15 minutes doesn’t give a lot of opportunity for anyone else to take a look at a PR.

File: README.md (hunk 1 above)
Comment (kenyon) [reply README.md:114]: Ah yeah, there are now three possible keys. #183 fixes this.

**Comment by ghoneycutt** [comment]
@alexjfisher apologies, i saw code that looked good with sufficient tests, so I merged :)

**Pull request merged**